util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,193 @@
+            "lower_limit": <value>,
+            "upper_limit": <value>,
+            "quantity": <value>
+          }
+
+    Returns:
+        A string representing the quantity range name to be used in the dcid.
+    """
+    lower_limit = match_dict["lower_limit"]
+    upper_limit = match_dict["upper_limit"]
+    quantity = match_dict["quantity"]
+
+    quantity = capitalize_word(quantity)
+    if upper_limit == "-":
+        return f"{lower_limit}OrMore{quantity}"
+
+    if lower_limit == "-":
+        return f"{upper_limit}OrLess{quantity}"
+
+    return f"{lower_limit}To{upper_limit}{quantity}"
+
+
+def generate_quantity_name(match_dict):
+    """Generate a name for a quantity.
+
+    Args:
+        match_dict: A dictionary containing quantity regex groups.
+          Expected syntax of match_dict
+          {
+            "value": <value>,
+            "quantity": <value>
+          }
+
+    Returns:
+        A string representing the quantity name to be used in the dcid.
+    """
+    value = match_dict["value"]
+    quantity = match_dict["quantity"]
+
+    quantity = capitalize_word(quantity)
+    return f"{value}{quantity}"
+
+
+def get_stat_var_dcid(stat_var_dict, ignore_props=None):
+    """Generates the dcid given a statistical variable.
+
+    Args:
+        stat_var_dict: A dictionary with property: values of the statistical
+          variable as key-value pairs.
+        ignore_props: A list of properties to ignore from stat_var_dict when
+          generating the dcid. This list of ignore_props will be added to the
+          default set of properties that are ignored. The ignore_props can be
+          used to account for dependent PVs to ignore when generating the dcid.
+          For example in the following statVar,
+          {
+            populationType: Person
+            measuredProperty: count
+            statType: measuredValue
+            healthInsurance: NoHealthInsurance
+            armedForceStatus: Civilian
+            institutionalization: USC_NonInstitutionalized
+          }
+          since the healthInsurance property indicates they are Civilian and
+          USC_NonInstitutionalized, ignore_props can be the list
+          ["armedForceStatus", "institutionalization"]. During the dcid
+          generation process, these PVs will not be considered.
+
+    Returns:
+        A string represting the dcid of the statistical variable.
+    """
+    # TODO: Allow support for values with namespaces (dcs: , schema:, ...)
+    # TODO: Add support for naming boolean constraints
+    # TODO: Stripping measurement properties (USC_) from constraints
+    # TODO: Replacing NAICS industry codes with the NAICS names
+    # TODO: Renaming cause of death properties
+    # TODO: Renaming DEA drug names
+    # TODO: Prepend or append text to some constraints to improve readability
+    # TODO: InsuredUmemploymentRate should become Rate_Insured_Unemployment
+
+    dcid_list = list()
+    denominator_suffix = ''
```

Comment:
use a consistent quote through the file, single or double.

convention seems to be to use single quotes in other files.